### PR TITLE
Add styles for reorder cue

### PR DESCRIPTION
**Pull request by joneff**
Related to https://github.com/telerik/kendo-theme-default/issues/600

### CODE REVIEW

File: scss/common/_dragclue.scss [hunk 1]
```diff
@@ -1,23 +0,0 @@
-@include exports( "common/dragclue" ) {
-
-    // Layout
-    .k-drag-clue {
```
Comment (gyoshev) [inline scss/common/_dragclue.scss:4]: These styles seem to be still used in the TreeList, TreeView, and Grid:

image

File: scss/grid.scss [hunk 2]
```diff
@@ -10,4 +10,4 @@
 
 // Component
 @import "grid/layout";
-@import "grid/theme";
+@import "grid/theme";
```
Comment (gyoshev) [inline scss/grid.scss:13]: The opposite is specified by the [.editorconfig file](https://github.com/telerik/kendo-theme-default/blob/develop/.editorconfig#L8). Make sure that any used editor [supports it or has a plugin installed](http://editorconfig.org/#download). The reasoning for having these is described in [this SO thread](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).

**Review by gyoshev — CHANGES_REQUESTED** [review]

File: scss/common/_dragclue.scss (hunk 1 above)
Comment (joneff) [reply scss/common/_dragclue.scss:4]: the styles are not removed. just singled out. They already exist in base: https://github.com/telerik/kendo-theme-default/blob/647f69552b6ebe8d4aecb6a484e8615ce19f11bc/scss/common/_base.scss#L596-L618

**Review by joneff — COMMENTED** [review]
The styles already exist in common/base

File: scss/grid.scss (hunk 2 above)
Comment (joneff) [reply scss/grid.scss:13]: As soon as I merge the other request, I will fix this one as well

**Review by gyoshev — APPROVED** [review]

**Pull request merged**